src/sql/workbench/api/browser/mainThreadConnectionManagement.ts

```diff
@@ -39,7 +40,7 @@ export class MainThreadConnectionManagement extends Disposable implements MainTh
 		}
 	}
 
-	public $registerConnectionEventListener(handle: number, providerId: string): void {
```

Comment:
the providerId parameter is not being used, I've tested with PG extension, and events are fired for it properly as well. since this is not an external facing API, I am removing it.